app/assets/stylesheets/modules/style.scss

```diff
@@ -83,8 +83,11 @@ a:hover {
 }
 
 form {
+  border-color: $red;
+  border-width: 1px;
```

Comment:
Properties should be ordered border-color, border-radius, border-width, color, margin, text-align, text-align